core/src/com/unciv/ui/worldscreen/AlertPopup.kt

```diff
@@ -247,8 +247,8 @@ class AlertPopup(val worldScreen: WorldScreen, val popupAlert: PopupAlert): Popu
         }
     }
 
-    override fun close() {
-        worldScreen.viewingCiv.popupAlerts.remove(popupAlert)
-        super.close()
-    }
+//    override fun close() {
+//        worldScreen.viewingCiv.popupAlerts.remove(popupAlert)
+//        super.close()
+//    }
```

Comment:
Commented out lines?

Reply:
Oh sorry - that defeat alert popping up twice... Wasn't sure whether treating popupAlerts as a _true_ queue would help, but no, it was actually added twice after all. Will need to revert this and that `first/pop` one. Or not?